_projects/2023/100407614/100407614.Rmd

```diff
@@ -0,0 +1,581 @@
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 3,
+    label = "15",
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 4,
+    label = "20",
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 5,
+    label = "25",
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 6,
+    label = "30",
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 7,
+    label = "35",
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 8,
+    label = "40",
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 9,
+    label = "45",
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 10,
+    label = "50",
+    vjust = -0.5, hjust = 1, size = 6,
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 11,
+    label = "55",
+    vjust = -0.5, hjust = 1, size = 6
+  )
+cross_chart
+
+```
+
+## Rescaling of age_males_death variable
+
+```{r}
+new_min <- 19.56
+new_max <- 55
+age_males_death <- cancer_deaths$age_males_death
+old_min <- min(age_males_death)
+old_max <- max(age_males_death)
+age_males_death <- (((age_males_death-old_min) / (old_max - old_min)) * (new_max - new_min)) + new_min
+cancer_deaths$age_males_death <- age_males_death
+
+```
+
+## Intersecting charts and filtering the data
+
+To overlay the line charts, we scale them by dividing the maximum value of sales_cig and age_males_death. As we can observe, there is a small peak in the raw data, specifically in the series representing cigarette sales, indicating that the authors who generated this graph likely underwent some standardization process to address it.
+
+```{r}
+scale <- max(sales_daily$sales_cig)/max(cancer_deaths$age_males_death)
+
+cancer_deaths <- cancer_deaths |> 
+  mutate(age_males_death = if_else(year >= 2016, NA, age_males_death))
+
+cross_chart <- ggplot() +
+  geom_line(data = sales_daily, aes(x = year, y = sales_cig, color = "Sales of Cigarettes"), size = 1) +
+  geom_line(data = cancer_deaths, aes(x = year, y = age_males_death * scale, color = "Cancer Deaths"), size = 1) +
+  labs(title = "Cigarette sales and lung cancer mortality in the US", x = "Year", y = "Cigarettes sold per adult per day") +
+  scale_color_manual(values = c("Sales of Cigarettes" = "#800080", "Cancer Deaths" = "#CC0000")) +
+  scale_y_continuous(sec.axis = sec_axis(~ ./scale, breaks = seq(0, 55, by = 5), name = "Rate of lung cancer deaths per 100,000 men", labels = scales::number_format()), breaks = seq(min(0), max(11), by = 1)) +
+  theme_minimal() +
+  theme(legend.position = "none")
+cross_chart
+
+```
+
+## Selecting the suitable font style
+
+```{r}
+sysfonts::font_add_google("Playfair Display", family="font_title")
+sysfonts::font_add_google("Lato", family="font_y_axis")
+showtext::showtext_auto()
```

Comment:
Instead of calling this manually, it's  better to add the option `fig.showtext=TRUE` to all chunks with a graph output.

Reply:
I don't fully understand this comment, as if I delete showtext::showtext_auto() and leave only sysfonts::font_add_google("Playfair Display") and sysfonts::font_add_google("Lato"), even if I put fig.showtext=TRUE in each chunk, the font doesn't change.